spring-cloud-stream/src/main/java/org/springframework/cloud/stream/endpoint/BindingsEndpoint.java

```diff
@@ -62,25 +53,41 @@ public BindingsEndpoint(List<InputBindingLifecycle> inputBindingLifecycles) {
 		this.objectMapper = new ObjectMapper();
 	}
 	
-	@ReadOperation
-	public List<?> bindings() {
-		return this.objectMapper.convertValue(this.gatherInputBindings(), List.class);	
-	}
-	
-	public StopEndpoint getStopEndpoint() {
-		return new StopEndpoint();
-	}
-	
-	public StartEndpoint getStartEndpoint() {
-		return new StartEndpoint();
+	@WriteOperation
+	public boolean changeState(@Selector String name, State state) {
```

Comment:
Returning a `boolean` strikes me as a little bit odd, particularly as it'll be `true` unless there's a bug and `state` is `null`.